install_scripts/templates/kubernetes/init.sh

```diff
@@ -262,7 +262,21 @@ initKube() {
 }
 
 isLatestKubernetes() {
```

Comment:
could this be renamed something like `shouldReinitK8s` now that it's doing something different?